src/FormatOrder.js

```diff
@@ -149,5 +154,6 @@ FormatOrder.prototype.ConvertSquareToSheet = function(txnMetadata, orderDetails,
  * @returns {string} formatted date
  */
 function convertISODate(date){
+// TODO: why isn't seconds here?
```

Comment:
My thought was to omit seconds here because when looking at arrival time, labeled time, closing time - I thought "5:53PM" would be enough. I'm open to other opinions though